content/webapp/components/IIIFItem/IIIFItem.tsx

```diff
@@ -84,9 +89,8 @@ const IIIFItem: FunctionComponent<ItemProps> = ({
           exclude={exclude}
         />
       );
-    case item.type === 'Sound' &&
-      !exclude.includes('Sound') &&
-      Boolean(item.id):
+    case (item.type === 'Sound' && !exclude.includes('Sound')) ||
+      (item.type === 'Audio' && !exclude.includes('Audio') && Boolean(item.id)):
```

Comment:
This is just me being unsure; should this be

```suggestion
    case ((item.type === 'Sound' && !exclude.includes('Sound')) ||
      (item.type === 'Audio' && !exclude.includes('Audio'))) && Boolean(item.id):
```

Reply:
yes, thanks, I blame my editor  😄